package.json

```diff
@@ -23,7 +23,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook:build": "build-storybook -c .storybook -o public/docs",
-    "deploy": "gatsby build && gh-pages -d public"
+    "deploy": "yarn build:external && cd public && surge"
```

Comment:
We should add in options for internal and external, I think most of the content the designers are working on is internal. 

so something like
`deploy:internal": "yarn build:nternal && cd public && surge"`

Reply:
Good point. I'll update the PR to include two options